src/tokenizers.js

```diff
@@ -987,7 +1555,10 @@ class PreTrainedTokenizer extends Callable {
         // Ensure it is less than model max length
         max_length = Math.min(max_length, this.model_max_length)
 
-        // TODO convert to tensor here?
+        // TODO: convert to tensor here?
+        /**
+         * @type {any}
+         */
```

Comment:
Converting to tensor looks like something to point out in an issue, if it is worth doing 
```suggestion
        /** @type {any[]} */
```